ui/src/server/services/userService.js

```diff
@@ -77,5 +81,26 @@ function refreshUserData(config, extServiceClient) {
         });
 }
 
+async function readUsersFileFromDisk(fileName) {
```

Comment:
couldn't find use of this function, should it be removed?

Reply:
it allows alternative way to load user file from disk, to be used by vzm-ui for example